.tina/collections/company.tsx

```diff
@@ -20,6 +21,21 @@ export const companySchema: Collection = {
     },
   },
   fields: [
+    {
+      type: "string",
+      name: "tip",
+      label: "Tip",
+      ui: {
+        component: ({}) => {
+          return (
+            <div>
+              Please fill out all the required labels before navigating to
```

Comment:
Make it clear there is a field at the bottom that needs to be filled out before saving.
This is a small PR, you should be able to get this checked by anyone on the team and get it merged asap